joomla/joomla-cms
# [4.0] [com_associations] Improving display of the sidebyside page

See discussion https://github.com/joomla/joomla-cms/pull/26945#issuecomment-555846578

### Summary of Changes
Moving the Target language field and buttons below the Target title preventing display of the language field with width to 100% until getting to mobile view.
This also aligns the item fields when using the same tab.
Correcting wrong js file name in the Association Field.
RTL css added


### Testing Instructions
Create a multilingual site using the multilingual sample data module
Display the com_associations side by side page.
Reduce window to mobile size

Switch to RTL and check again

### Before patch

![sidebysidebefore](https://user-images.githubusercontent.com/869724/69319182-fcfa7700-0c3e-11ea-871b-ad5712802005.gif)

### After patch

![sidebysideafter](https://user-images.githubusercontent.com/869724/69319669-ec96cc00-0c3f-11ea-8bbb-f2de0db5092d.gif)


### Important Note
Independent from this PR.
The Clear button is not acting as it correctly does in 3.x.
It should not only clear the existing association itself in the Reference but also reload the Target with empty item fields.
@dgrammatiko @Fedik
